init/haulagedb.sh

```diff
@@ -0,0 +1,115 @@
+#!/bin/bash
```

Comment:
I'm curious why the shell script that essentially passes through to python? Is it just carryover from a time before when there was just a shell script? Does the shell wrapper do something that I am missing?